jscomp/syntax/src/res_printer.ml

```diff
@@ -386,9 +386,9 @@ let classifyIdentContent ?(allowUident = false) ?(allowHyphen = false) txt =
       if i == len then NormalIdent
       else if i == 0 then
         match String.unsafe_get txt i with
+        | '\\' -> UppercaseExoticIdent
         | 'A' .. 'Z' when allowUident -> loop (i + 1)
         | 'a' .. 'z' | '_' -> loop (i + 1)
-        | '-' when allowHyphen -> loop (i + 1)
```

Comment:
why is this line removed?

Reply:
Oh thanks, I never intended to do this. (but tests are fine?)

Reply:
But, it seems we don't actually need this line because the scanner never makes an unwrapped ident with `^-.*` I guess